test/remote/gateways/remote_litle_test.rb

```diff
@@ -218,6 +229,12 @@ def test_successful_purchase_with_android_pay
     assert_equal 'Approved', response.message
   end
 
+  def test_successful_purchase_with_google_pay
```

Comment:
💬  Comment:

I not inherently bad, but I think that may this test is unnecessary, the change that this PR introduces doesn't change the current behavior against the API, it just deals with a particular issue with CORE.

Reply:
I think this is ok, there is actually functionality added (accepting `google_pay` as input), and should be tested.